docs/abi.md

```diff
@@ -237,11 +237,13 @@ If creating the specified Node would violate
 [information flow control](/docs/concepts.md#labels), returns
 `ERR_PERMISSION_DENIED`.
 
-- `param[0]: usize`: Source buffer holding serialized `NodeConfiguration`
-- `param[1]: usize`: Serialized NodeConfiguration size in bytes
-- `param[2]: usize`: Source buffer holding serialized `Label`
-- `param[3]: usize`: Label size in bytes
-- `param[4]: usize`: Handle to channel
+- `param[0]: usize`: Source buffer holding the UTF-8 encoded name
+- `param[1]: usize`: Name size in bytes
```

Comment:
(After the fact) It would be nice to have something in the text above this that explains what the "name" is &ndash; IIUC it's a name that doesn't need to match anything else (neither the module name nor the entrypoint name of the `NodeConfiguration`) and doesn't need to be unique, and which is only used for debugging.  Is that right?